docs/src/main/sphinx/release/release-356.md

```diff
@@ -0,0 +1,80 @@
+# Release 356 (XX Apr 2021)
+
+## General
+
+* Add support for {doc}`/sql/match-recognize`. ({issue}`6111`)
+* Add {func}`soundex` function. ({issue}`4022`)  
+* Introduce `system.metadata.materialized_view_properties` table for listing available materialized view properties. ({issue}`7615`)
+* Introduce planning time limit which can be specified using `query.max-planning-time` config property. ({issue}`7213`)
+* Add configuration for the query info sent to clients.  This allows clients to be directed to a different location for 
+  information about their query.  This is set using the `query.info-url-template` property.  ({issue}`7678`)
+* Improve cancellation of queries during planning phase. ({issue}`7213`)
+* Improve performance of `ORDER BY ... LIMIT` queries over a `LEFT JOIN`. ({issue}`7028`)
+* Improve performance of queries with predicates on boolean columns. ({issue}`7263`)
+* Improve planning time for queries with large `IN` predicates. ({issue}`7556`)
+* Improve performance of queries that contain join on `VARCHAR` keys of different length. ({issue}`7644`)
+* Improve performance of queries with `late_materialization` enabled. ({issue}`7695`)
+* Reduce coordinator network overhead when scheduling queries. ({issue}`7351`)
+* Fix possible deadlock for `JOIN` queries when spilling is enabled. ({issue}`7455`)
+* Fix incorrect results for queries with full join which has scalar (left or right) input. ({issue}`7629`)
+* Add dedicated access control for creating and dropping materialized views. ({issue}`7645`)
+
+## JDBC driver
+
+* Add `externalAuthenticationTokenCache=MEMORY` property so Oauth2 credentials are cached in memory. Thanks to this user can avoid unnecessary authentication flow. ({issue}`7309`)
```

Comment:
How about:

> Add support for caching OAuth2 credentials in memory to avoid unnecessary authentication flows.

Generally, we want to explain what's the impact of the change.

Reply:
People generally want to know how to enable the change. So if we drop the config property we should still link to the config property somehow (unless it's enabled by default).